src/api/models.py

```diff
@@ -61,13 +61,6 @@ class CompareFileUpload(models.Model):
     rfilename = models.CharField(max_length=32,null=False,blank=False)
     status = models.IntegerField(default=200,blank=False)
 
-class CheckLicenseFileUpload(models.Model):
```

Comment:
This is being used in some of the tests as well. You need to remove the usage for this from there as well (tests.py)

Reply:
I added issue #422 to track fixing the unit tests.  Since the unit tests are already failing for other reasons, I'll go ahead and merge this PR and hopefully fix the unit tests in an upcoming release.